easybuild/framework/easyconfig/tweak.py

```diff
@@ -357,14 +358,7 @@ def __repr__(self):
         _log.debug("Generated file name for tweaked easyconfig file: %s", tweaked_ec)
 
     # write out tweaked easyconfig file
-    if os.path.exists(tweaked_ec):
-        if build_option('force'):
-            print_warning("Overwriting existing file at %s with tweaked easyconfig file (due to --force)", tweaked_ec)
-        else:
-            raise EasyBuildError("A file already exists at %s where tweaked easyconfig file would be written",
-                                 tweaked_ec)
-
-    write_file(tweaked_ec, ectxt)
+    write_file(tweaked_ec, ectxt, backup=True, overwrite=False, verbose=True)
```

Comment:
Ah ok, I understand it now but the kw name is not very helpful, how about `unforced_overwrite`?

Reply:
`overwrite=False` means "don't overwrite unless `--force` is used"

Without also adding `backup=True`, you may overwrite when using `--force` without backing up the file.

Whether you want a backup or not depends on in what context you're calling `write_file`.

Reply:
Ok, but the kw name needs tweaking, it's not unreasonable to expect `overwrite=False` not to overwrite.

Reply:
I had `overwrite_without_force` before, which I found too long, but would that be better?

Reply:
how about `always_overwrite` ?

Reply:
`overwrite_without_force` is long but since the option is so connected to `--force` I think it is worth the keystrokes, it will save people having to read the docstring

Reply:
@mboisson `always_overwrite` makes sense to me. Can you make that change?

Reply:
I'm kind of lost in all of the changes you PR'd to us now :/

Reply:
I *think* I did it.